cypress/e2e/test.js

```diff
@@ -0,0 +1,34 @@
+import { Account } from "./components/accounts";
```

Comment:
We don't need to commit this file as code is not completed fully

Reply:
Ok .